src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -179,6 +179,7 @@ const ActiveState = ({
                                             can_reply: getProp(item.permissions, 'can_reply', false),
                                             can_resolve: getProp(item.permissions, 'can_resolve', false),
                                         }}
+                                        allowCollapse
```

Comment:
Is this desired? It will immediately reflect in the live Preview experience for long comments.

Reply:
Without it the top-level comments won't be collapsible. Is there a check that I can put inside so that it will be true only for Threaded Comments? like:
```suggestion
                                        allowCollapse={hasReplies}
```
?

I'll bring it to the team tomorrow.

Reply:
@kkuliczkowski-box suggested using features for this, that way it would be enabled for Canvas only for now.
Similarly to this
https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/ActivitySidebar.js#L892

I found that I can use https://github.com/box/box-ui-elements/blob/master/src/elements/common/feature-checking/FeatureContext.js

I decided to put that in single place in `Comment.js`. What do you think?
I have implementation ready, but still some tests are failing before I commit.

Reply:
here's the commit https://github.com/box/box-ui-elements/pull/3079/commits/f807afc79df5c9a09992de997043b61474462183